governance-exercises/Poll Exercise 03.md

```diff
@@ -0,0 +1,110 @@
+
+(Generally, Signal Requests are [no longer permitted](https://mips.makerdao.com/mips/details/MIP4c2SP27#amended-mips-and-components), unless specifically triggered and required by an existing MIP, such as MIP24.)
+
+### Problems with the Standing Orders (aka Instruction Sets)
+
+I assume the Exercises' Standing Orders are equivalent to the "Instruction Sets" of current RWA structures. (See [1](https://forum.makerdao.com/t/project-andromeda-risk-legal-assessment/20969#instruction-sets-9), [2](https://forum.makerdao.com/t/clydesdale-vault-hq/17923#clydesdale-instruction-sets-4).) For the purpose of the Exercise, I use [Monetalis Clydesdale](http://forum.makerdao.com/t/mip65-clydesdale-governance-framework-setup/16565)'s governance framework as a reference point in terms of best practices/prerequisites concerning a Standing Order/Instruction Set:
+
+> One can somewhat see the Instructions coming as a Governance Poll outcome from Maker as the “code” and the trust/trustee/3rd parties as the “machine” that safely executes the code and puts relevant data back to Maker. ...**These instruction sets are set out to be as detailed as possible** - including **clear specification** of checks and validations required along the way.
+>
+> Maker can structure instruction sets in any manner it prefers and submit via Governance Poll to the Trust - and given it is legal and possible - it will execute. ...
+>
+> As a safety measure we have also developed a library of key instruction sets that address key situations and can be put up as Governance Polls, **that we know are appropriately designed and confirmed executables.** This way most eventualities are planned for and can be acted upon confidently and speedily by Maker. ...
+>
+> **Trustee has no discretion for executions outside of Governance Poll approved instruction sets.** [emphasis added]
+
+There are three problems related to the Standing Orders in the Exercise Prompt, which run counter to both aspects of my mandate as a Governance Facilitator.
+
+First, the proposed Standing Order does not offer voters enough information to make an informed decision; this is due to the ambiguities in the proposed Standing Order, which I assume are non-trivial. If the Standing Order/Instruction Set is to be akin to safely executable code, it cannot be ambiguous without presenting significant risk to Maker. 
```

Comment:
Eh, I mean it _can_ be ambiguous without presenting risk. It's still bad though, as different parties will end up arguing about how it should be interpreted.